### DISCUSSION TITLE
Contributing to the project

### DISCUSSION THREAD

**Discussion by mghildiy** [grafana/tempo#2251] [General]
Hi,

I am Manish, a Java developer.
I would like to contribute to this project.
Where should I start looking into?

**Comment by mapno** [comment]
Hi, welcome!

You can explore our [issues](https://github.com/grafana/tempo/issues) and pick anyone you like. Some are tagged with [`good-first-issue`](https://github.com/grafana/tempo/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22), which we think are specially good for newcomers.
